### PR TITLE
(node/lsstcam-vw03.ls.lsst.org) provision host as EL9 & add test

### PR DESCRIPTION
**Pull request by badenerb**

### CODE REVIEW

File: spec/hosts/nodes/lsstcam-vw03.ls.lsst.org_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'lsstcan-vw03.ls.lsst.org', :sitepp do
```
Comment (jhoblitt) [inline spec/hosts/nodes/lsstcam-vw03.ls.lsst.org_spec.rb]:
```suggestion
describe 'lsstcam-vw03.ls.lsst.org', :sitepp do
```

**Review by jhoblitt — DISMISSED** [review]
Thank you for adding this host to ipam.

File: spec/hosts/nodes/lsstcam-vw03.ls.lsst.org_spec.rb (hunk 1 above)
Comment (badenerb) [reply spec/hosts/nodes/lsstcam-vw03.ls.lsst.org_spec.rb]: Thanks! The n and m keys are pretty close! 🤣

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Pull request merged**